org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/manifest.xml

```diff
@@ -12,9 +12,24 @@
       <schema id="oic.core" type="OCF" />
       <schema id="oic.baseresource" type="OCF" />
   </schemas>
-  <!-- associated onboarding module -->
   <onboarding id="org.opent2t.onboarding.nest">
```

Comment:
><onboarding id="org.opent2t.onboarding.nest"> [](start = 2, length = 45)

This whole section was removed from the Wink thermostat manifest. Why is it required for nest but not for Wink?